environment.yml

```diff
@@ -24,3 +24,11 @@ dependencies:
   - bioconda::bedtools=2.29.2
   - bioconda::ucsc-bedgraphtobigwig=357
   - bioconda::ucsc-bedtobigbed=357
+  
+  - r-base=4.0.1
+  - r-devtools=2.3.0
+  - bioconductor-deseq2=1.28.0
+  - bioconductor-drimseq=1.16.0
+  - bioconductor-dexseq=1.34.0
+  - bioconductor-stager=1.10.0
+  - bioconductor-bsgenome=1.56.0
```

Comment:
```suggestion
  - conda-forge::r-base=4.0.1
  - conda-forge::r-devtools=2.3.0
  - bioconda::bioconductor-deseq2=1.28.0
  - bioconda::bioconductor-drimseq=1.16.0
  - bioconda::bioconductor-dexseq=1.34.0
  - bioconda::bioconductor-stager=1.10.0
  - bioconda::bioconductor-bsgenome=1.56.0
```

Reply:
I have created a separate section for `conda-forge` packages at the top of the file.

Reply:
conda-forge and bioconda packages are now separated.